server/sched_test.go

```diff
@@ -102,10 +106,16 @@ type reqBundle struct {
 	ggml    *llm.GGML
 }
 
-func (scenario *reqBundle) newServer(gpus gpu.GpuInfoList, model string, ggml *llm.GGML, adapters []string, projectors []string, opts api.Options, numParallel int) (llm.LlamaServer, error) {
+func (scenario *reqBundle) newServer(payloadFS fs.FS, gpus gpu.GpuInfoList, model string, ggml *llm.GGML, adapters []string, projectors []string, opts api.Options, numParallel int) (llm.LlamaServer, error) {
 	return scenario.srv, nil
 }
 
+func dummyPayloads() fs.FS {
```

Comment:
```suggestion
func testFS() fs.FS {
```

Reply:
I think this can go away entirely with the refactoring suggested above.